lxd/instances_post.go

```diff
@@ -416,6 +417,164 @@ func createFromMigration(s *state.State, r *http.Request, projectName string, pr
 	return operations.OperationResponse(op)
 }
 
+// createFromConversion uploads the image to the server's backup directory, converts it into the
```

Comment:
Is this comment still correct? It doesn't always do this anymore right?

Reply:
Thanks. Fixed